source/admin_manual/list_of_events.rst

```diff
@@ -1574,6 +1597,179 @@ renumbering was done after all.
++-----------------------+------------------------------------------------------+
+
+metacache_user_bundle_download_started
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+metacache_user_bundle_download_finished
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+metacache_mailbox_bundle_download_started
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+metacache_mailbox_bundle_download_finished
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+User or mailbox index bundle file is downloaded. These events can happen while
+the user or mailbox is being refreshed.
+
++-----------------------+------------------------------------------------------+
+| Field                 | Description                                          |
++=======================+======================================================+
+| fname                 | Bundle filename                                      |
++-----------------------+------------------------------------------------------+
+| bundle_type           | Bundle type: diff, base or self                      |
++-----------------------+------------------------------------------------------+
+| bundle_size           | Size of the bundle file in bytes (uncompressed)      |
++-----------------------+------------------------------------------------------+
+| error                 | Error message if the download failed                 |
++-----------------------+------------------------------------------------------+
+
+metacache_upload_started
+^^^^^^^^^^^^^^^^^^^^^^^^
+metacache_upload_finished
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Changes in metacache are being uploaded to storage.
+
++-----------------------+------------------------------------------------------+
+| Field                 | Description                                          |
++=======================+======================================================+
+| error                 | Error message if the upload failed                   |
++-----------------------+------------------------------------------------------+
+
+metacache_user_bundle_upload_started
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+metacache_user_bundle_upload_finished
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+metacache_mailbox_bundle_upload_started
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+metacache_mailbox_bundle_upload_finished
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+User or mailbox index bundle file is uploaded. These events can happen while
+the user or mailbox is being uploaded. Note that the metacache_user_* events
+can also be inherited from a mailbox event and include the mailbox fields if
+the user upload was triggered by a mailbox upload.
+
++-----------------------+------------------------------------------------------+
+| Field                 | Description                                          |
++=======================+======================================================+
+| fname                 | Bundle filename                                      |
```

Comment:
i know the code is probably already committed but is there a reason we can't use "filename" here instead?